core/vertx/src/test/java/io/thorntail/vertx/VertxTest.java

```diff
@@ -0,0 +1,25 @@
+package io.thorntail.vertx;
+
+import javax.inject.Inject;
+
+import io.thorntail.test.ThorntailTestRunner;
+import io.vertx.core.Vertx;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by bob on 6/22/18.
+ */
+@RunWith(ThorntailTestRunner.class)
+public class VertxTest {
```

Comment:
We already test `Vertx` injection here:
https://github.com/thorntail/thorntail/blob/4.x/testsuite/simple/vertx/src/test/java/io/thorntail/testsuite/vertx/Emitter.java#L16 
and here:
https://github.com/thorntail/thorntail/blob/4.x/testsuite/simple/vertx/src/test/java/io/thorntail/testsuite/vertx/VertxInitializerTest.java#L49